api/TextDecoder.json

```diff
@@ -116,9 +116,15 @@
             "ie": {
               "version_added": false
             },
-            "nodejs": {
-              "version_added": null
-            },
+            "nodejs": [
+              {
+                "version_added": "11.0.0"
+              },
+              {
+                "version_added": "8.3.0",
+                "notes": "Exported from the <code>util</code> module but not globally available."
```

Comment:
```suggestion
                "notes": "Available as a part of the <code>util</code> module."
```

Reply:
I realized this note was copied from that of the rest of `TextDecoder` and `TextEncoder`.  I'm going to open a PR to update this in the other locations!

Reply:
It looks like the parent feature also uses `partial_implementation`. I'm wondering if the right thing to do here is to do the same as the parent, but revisit this to use `alternative_name` instead, which I think would more accurately capture what's happening.

Reply:
(I'll defer to @queengooborg whether the current changes satisfy though. A little _more_ inconsistency is probably not a huge problem here.)

Reply:
We should certainly update the child features to mark them as `partial_implementation`, but I think we can perform that in a separate PR!